app/build.gradle

```diff
@@ -92,9 +92,9 @@ android {
 dependencies {
 
     // At a time only one WebexSDK should be used.
-    implementation 'com.ciscowebex:webexsdk:3.9.1' // For full flavor
-//    implementation 'com.ciscowebex:webexsdk-wxc:3.9.1' //For webexCalling flavor
-    //implementation 'com.ciscowebex:webexsdk-meeting:3.9.1' // For meeting flavor
+    implementation 'com.ciscowebex:webexsdk:3.9.1.3-SNAPSHOT' //For webexCalling flavor
+    //implementation 'com.ciscowebex:webexsdk-meeting:3.9.0' // For meeting flavor
+    //implementation 'com.ciscowebex:webexsdk:3.9.0' // For full flavor
```

Comment:
This we can update to 3.9.1.3-SNAPSHOT and for webexCalling and meeting flavor. Basically usage of all 3 flavors should be demoed.

Reply:
done